third_party/ducc/BUILD

```diff
@@ -0,0 +1,30 @@
+licenses(["notice"])
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
```

Comment:
Even in the current implementation without multithreading, you need to configure DUCC _not_ to use `std::mutex` / `std::thread`.

With the version you currently have being imported, you at least need to define `DUCC0_NO_THREADING` and ` DUCC0_NO_FFT_CACHE`.
- The `DUCC0_NO_THREADING` is needed to prevent explicit use of `std::mutex`, since we need to use `tsl::mutex` to be compatible with all platforms.  They are not always the same thing.
- The `DUCC0_NO_FFT_CACHE` is needed to avoid race conditions when threading is disabled

With newer DUCC, you need to instead
- define `DUCC0_CUSTOM_LOWLEVEL_THREADING`
- create a `ducc0_custom_lowlevel_threading.h` file that sets up the configuration and can be found on the include path (i.e. you would put it here in `third_party/ducc` and add it to the set of `includes`.

Reply:
I've set to `DUCC0_NO_LOWLEVEL_THREADING` for now. We will handle multi-threaded in subsequent PR.